routing_model/src/scenario_tests.rs

```diff
@@ -175,6 +176,7 @@ fn run_test(test_name: &str, start_state: &State, events: &[Event], expected_sta
         },
         final_state.failure,
     );
+    dbg!(&final_state);
```

Comment:
Why did you need this? did the assert_eq not provide you the needed info?
If so, we should consider making sure it does.

Reply:
Well it's the same info, but dbg gives it with linebreaks and indentation. I struggle to parse it when it's just one long long line :(

Reply:
https://crates.io/crates/pretty-assertions ?